milksnake/empty.c

```diff
@@ -0,0 +1,2 @@
+void init_native__lib() {}
+void PyInit__native__lib() {}
```

Comment:
this only works for the current `example/setup.py` because there cffi `module_path` name was `_native`.
If one uses a different name, it won't work. Maybe `empty.c` should be a template wher you interpolate the string of the module name.

Reply:
Makes sense, I didn't look much into the naming here. Will fix it.

Reply:
Thanks!